resources/[core]/customization/src/client/misc/events.ts

```diff
@@ -17,6 +17,22 @@ onUI('customization.choose-gender', () => {
   creationManager.chooseGender();
 });
 
+onUI('customization.set-information-firstname', (firstname) => {
+  creationManager.setFirstName(firstname);
+});
+
+onUI('customization.set-information-lastname', (lastname) => {
+  creationManager.setLastName(lastname);
+});
+
+onUI('customization.set-information-dateofbirth', (dateOfBirth) => {
```

Comment:
this will not come through as a date. it will need to be converted to a string of some sort probably ISO string and converted back into a date object before sending to creationManager.